packages-exp/auth-exp/index.ts

```diff
@@ -60,8 +65,11 @@ export { browserPopupRedirectResolver } from './src/platform_browser/popup_redir
 // MFA
 export { PhoneMultiFactorGenerator } from './src/platform_browser/mfa/assertions/phone';
 
-export const initializeAuth = _initializeAuthForClientPlatform(
-  ClientPlatform.BROWSER
-);
+export function getAuth(app?: FirebaseApp): Auth {
+  return initializeAuth(app, {
+    popupRedirectResolver: browserPopupRedirectResolver,
+    persistence: [indexedDBLocalPersistence, browserLocalPersistence]
```

Comment:
why can't we do the same style of fallback for workers?

Reply:
The "fallback" is doing different things. Here, it's saying "check these persistences in this order," whereas for workers, it's literally checking to see if it's available (if you tried to pass indexedDB into it when it wasn't available, the instantiation would just error out)

Reply:
Wouldn't auth fallback to memory persistence when indexedDB is not available. Why is it different for workers?

Reply:
Currently it would error. It probably shouldn't but we need to clarify that separately. For now I'd like to move ahead and deal with that issue in a separate PR. I'm tracking this in b/165376704